managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -242,11 +242,13 @@ ManagedCursor openCursor(String name, InitialPosition initialPosition) throws In
      *             operation will trigger the creation of the cursor.
      * @param cursorProperties
      *            the properties for the Cursor
+     * @param inclusive
+     *            whether to read from the specified position
      * @return the ManagedCursor
      * @throws ManagedLedgerException
      */
     ManagedCursor openCursor(String name, InitialPosition initialPosition, Map<String, Long> properties,
```

Comment:
Could you add default override methods for compatibility? Because they are common APIs that can be used by other components like protocol handler.

e.g.

```java
    default void asyncOpenCursor(String name, InitialPosition initialPosition, Map<String, Long> properties,
                         Map<String, String> cursorProperties, OpenCursorCallback callback, Object ctx) {
        asyncOpenCursor(name, initialPosition, properties, cursorProperties, false, callback, ctx);
    }
```